book/content/disabling.md

```diff
@@ -0,0 +1,23 @@
+# Disabling a Stage
+
+
+A stage can be disabled by specifiying a boolean flag in the `using` section of the mro. For example:
+
+```mro
+stage MY_STAGE(
+...
+) using (
+    disabled  = self.no_bam,
```

Comment:
Err, this would not be legal mro syntax.  You can't disable a stage, only a `call`.

Reply:
oh doh, that's right..hmm